dev_tools/shell_tools.py

```diff
@@ -118,6 +118,53 @@ def abbreviate_command_arguments_after_switches(cmd: Tuple[str, ...]) -> Tuple[s
     return tuple(result)
 
 
+def run(
+    args: Union[str, List[str]],
+    *,
+    log_run_to_stderr: bool = True,
+    abbreviate_non_option_arguments: bool = False,
+    check: bool = True,
+    text: bool = True,
+    **subprocess_run_kwargs,
+) -> subprocess.CompletedProcess:
+    """Call subprocess.run with an option to log executed command to stderr.
+
```

Comment:
maybe something to distinguish why you would use this and not run_cmd below?

Reply:
Ack.  I actually plan to remove run_cmd and run_shell in a followup PR and only use the `run()` function.
This PR is 1 of 4 for [#4394](https://github.com/quantumlib/Cirq/issues/4394#issuecomment-1155868381).

Thanks for quick review.